### PR TITLE
Add basic stringtable implementation

### PR DESCRIPTION
**Pull request by Pi123263**
Adds a basic implementation of localization with strings mainly for the SafeStart system. 
ATTENTION: This commit only creates the strintable and strings, it does not actually add then into the SafeStart code!

### CODE REVIEW

File: stringtables/COAL_localization.st [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+   Target_en_us "[CRF] : %1 Has Readied Up a faction"
+   Target_de_de "[CRF] : %1 hat eine Fraktion für bereit erklärt"
+   Comment "Generic version of ready string in case a certain side/faction is not represented in the other strings"
+   Modified 1626905909
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0C5CAEB3E}" {
+   Id "#Coal_SS_Faction_Readied_Blufor"
+   Target_en_us "[CRF] : %1 Has Readied Up Blufor"
```
Comment (tanaaKa) [inline stringtables/COAL_localization.st]: Lets change this verbiage while we're here. I'm thinking

`Side READY - name`

Example:
BLUFOR READY - tanaKa

File: stringtables/COAL_localization.st [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+   Target_de_de "[CRF] : %1 hat Blufor für bereit erklärt"
+   Modified 1626905777
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0D88FF976}" {
+   Id "#Coal_SS_Faction_Readied_Indfor"
+   Target_en_us "[CRF] : %1 Has Readied Up Indfor"
```
Comment (tanaaKa) [inline stringtables/COAL_localization.st]: INDFOR READY - %1

File: stringtables/COAL_localization.st [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+   Target_de_de "[CRF] : %1 hat Indfor für bereit erklärt"
+   Modified 1626905782
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0DB27AEB5}" {
+   Id "#Coal_SS_Faction_Readied_Opfor"
+   Target_en_us "[CRF] : %1 Has Readied Up Opfor"
```
Comment (tanaaKa) [inline stringtables/COAL_localization.st]: OPFOR READY - %1

File: stringtables/COAL_localization.st [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+   Target_en_us "[CRF] : %1 Has Unreadied a faction"
+   Target_de_de "[CRF] : %1 hat eine Fraktion für nicht bereit erklärt"
+   Comment "Generic version of unreadied string in case a certain side/faction is not represented in the other strings"
+   Modified 1626905895
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0C8E44A59}" {
+   Id "#Coal_SS_Faction_Unreadied_Blufor"
+   Target_en_us "[CRF] : %1 Has Unreadied Blufor"
```
Comment (tanaaKa) [inline stringtables/COAL_localization.st]: BLUFOR Not Ready - %1

File: stringtables/COAL_localization.st [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+   Target_de_de "[CRF] : %1 hat Blufor für nicht bereit erklärt"
+   Modified 1626905824
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0CF9391A8}" {
+   Id "#Coal_SS_Faction_Unreadied_Indfor"
+   Target_en_us "[CRF] : %1 Has Unreadied Indfor"
```
Comment (tanaaKa) [inline stringtables/COAL_localization.st]: INDFOR Not Ready - %1

File: stringtables/COAL_localization.st [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+   Target_de_de "[CRF] : %1 hat Indfor für nicht bereit erklärt"
+   Modified 1626905831
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0CE385C91}" {
+   Id "#Coal_SS_Faction_Unreadied_Opfor"
+   Target_en_us "[CRF] : %1 Has Unreadied Opfor"
```
Comment (tanaaKa) [inline stringtables/COAL_localization.st]: OPFOR Not Ready - %1

**Review by tanaaKa — CHANGES_REQUESTED** [review]

File: stringtables/COAL_localization.st (hunk 1 above)
Comment (Pi123263) [reply stringtables/COAL_localization.st]: Sounds good, I assume for a generic version it'd just be "FACTION READY - name" then?
That'd be my first thought at least

Comment (Pi123263) [reply stringtables/COAL_localization.st]: Also keeping the CRF tag for ease of recognition/debugging I assume?

**Pull request merged**